### ISSUE TITLE
forward XML to client

### ISSUE THREAD

**Issue by mumem** [MUME/MMapper#285]
Currently the XML targs ("change xml on") are not forwarded to the client. It would be great if this would be configurable in MMapper via the preferences dialog, so that also clients (like Mudlet) can receive these XML codes.

**Comment by nschimme** [comment]
Not discounting this ticket, but couldn't Mudlet use GMCP instead to get some of the structured information?

**Comment by MuMeM** [comment]
Yes, some. But not all. E.g. the <character> and <enemy> information is not sent, and all the combat messages (<hit>, <damage>, <avoid_damage>. XML makes it **so much easier** to identify all this information, without resorting to parsing.
